test/reactionsystem_structure/higher_order_reactions.jl

```diff
@@ -51,7 +51,10 @@ let
 end
 
 # Tests that the discrete jump systems are equal.
-let
+# Currently fails because binomial only takes Int input (and X is Float64).
+# I see several solutions, but depends on whether we allow species to be specified as Int64.
+# I have marked this one as broken for now.
+@test_broken let 
```

Comment:
This test will remain broken until we decide how to handle non-Float64 definitions of species. Which will be messy, because if we do we have to rewrite the `@compound` stuff to permit that as well.

Once sorted out, we can fix these tests using the best solution (which depends on whether that is allowed or not).